### ISSUE TITLE
Help Panel: the initial focus should be on the text, not on the Search Box

### ISSUE THREAD

**Issue by ecarlevaro** [rstudio/rstudio#10600]
When moving to the Help Panel using **CTRL+3**, the cursor focus is on the Search box on the top right. This makes it impossible to scroll down the help page unless using the mouse to switch focus to the page.

The initial focus after CTRL+3 should be on the page itself and only if the user desires to search can hit CTRL+F. Another more cumbersome solution is to have another shortcut to move focus from the Search box to the page.

Using Rstudio 2021.09.1 Build 372 on Windows 10

Related to #5149 #6108

**Comment by ronblum** [comment]
Verified in
- RStudio Desktop 2023.05.0+92 on MacOS 13.2
- RStudio Desktop 2023.05.0+92 on Windows 11
- RStudio Desktop 2023.05.0+92 on Ubuntu 20.4.5 via Chrome on MacOS 13.2
